ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino

```diff
@@ -581,17 +594,17 @@ void setup() {
 
   server.on("/api/printer", HTTP_GET, [](AsyncWebServerRequest * request) {
     // https://docs.octoprint.org/en/master/api/printer.html#retrieve-the-current-printer-state
-    String sdReadyState = String(storageFS.activeSD() ? "true" : "false");  //  This should request SD status to the printer
-    String readyState = String(printerConnected ? "true" : "false");
+    String sdReadyState = stringify(storageFS.activeSD());   //  This should request SD status to the printer
```

Comment:
This should ask the printer for it's SD card, not critical for now

Reply:
I think SD on printer will never be used so maybe a fixed false may be correct